NameLayerMain/src/vg/civcraft/mc/namelayer/database/GroupManagerDao.java

```diff
@@ -981,6 +983,37 @@ public synchronized void removeGroupInvitation(UUID uuid, String groupName){
 		}
 	}
 	
+	
+	/**
+	 * Use this method to load a specific invitation to a group without the notification. 
+	 * @param playerUUID The uuid of the invited player.
+	 * @param group The group the player was invited to. 
+	 */
+	public synchronized void loadGropInvitation(UUID playerUUID, Group group){
+		try {
+			loadGroupInvitation.setString(1, playerUUID.toString());
+			loadGroupInvitation.setString(2, group.getName());
```

Comment:
do you group null test at the top of the method or at the very least before this line.